Adjust references to magic(4) to be magic(5) in the mkisofs/mkisofs.8 man page.

This patch should be sent upstream.

--- cdrtools-3.01/mkisofs/mkisofs.8.orig	2016-04-20 07:34:14.517901344 -0700
+++ cdrtools-3.01/mkisofs/mkisofs.8	2016-04-20 07:35:06.243987484 -0700
@@ -2674,7 +2674,7 @@
 The format of the
 .I magic
 file is almost identical to the
-.BR magic (4)
+.BR magic (5)
 file used by the Linux
 .BR file (1)
 command - the routines for reading and decoding the
@@ -2713,7 +2713,7 @@
 .TE
 .PP
 The format of the file is described in the
-.BR magic (4)
+.BR magic (5)
 man page. The only difference here is that for each entry in the magic file, the
 .I message
 for the initial offset
